docs/pages/configuration/_partials/functions/ensure_pull_secrets/example.mdx

```diff
@@ -0,0 +1,20 @@
+#### Example:
+
+```yaml
+images:
+  myImage:
+    dockerfile: ./Dockerfile
+
+pipelines:
+  pull_secrets: |-
+     echo "Welcome to your custom pull_secrets pipeline!"
+
+     # Exchange Dockerfile here (use --set-string if you want to force the field to be a string e.g. when setting labels to "true")
+     pull_secrets myImage --set dockerfile=./Dockerfile.dev --set-string key1=value1
```

Comment:
the function is called ensure_pull_secrets and use an actual pull secret here instead of an image